low-level/files/opt/seagate/sspl/conf/sspl.conf.LR2.yaml

```diff
@@ -184,13 +185,18 @@ SASPORTSENSOR:
 MEMFAULTSENSOR:
    monitor: true
    threaded: true
-   probe: sysfs
+   probe: procfs
```

Comment:
Why it is changed from sysfs to procfs?

Reply:
This sensor is failing due to 'SysFS' not having meminfo.
Actually meminfo can be found under procfs, not under sysfs. I don't know how author tested it.
`
[root@smc60-m11 ~]# ls -lrt /proc/meminfo
-r--r--r-- 1 root root 0 Dec 23 00:40 /proc/meminfo
`

`
2021-12-23 00:37:04 sspl [18089]: DEBUG [check_and_conclude_initialization] Begin MemFaultSensor.conclude_initialization()
2021-12-23 00:37:05 sspl [18089]: ERROR [execute_thread] MemFaultSensor, 'SysFS' object has no attribute 'get_proc_memory'
  File "/opt/seagate/cortx/sspl/low-level/sensors/impl/generic/node_memory_fault.py", line 147, in run
    mem_path = self._utility_instance.get_proc_memory('meminfo')
AttributeError: 'SysFS' object has no attribute 'get_proc_memory'
  File "/opt/seagate/cortx/sspl/low-level/sensors/impl/generic/node_memory_fault.py", line 147, in run
    mem_path = self._utility_instance.get_proc_memory('meminfo')
AttributeError: 'SysFS' object has no attribute 'get_proc_memory'
2021-12-23 00:37:05 sspl [18089]: CRITICAL [execute_thread] MemFaultSensor is stopped and unrecoverable. MemFaultSensor, 'SysFS' object has no attribute 'get_proc_memory'. Impact: Server memory usage can not be monitored. Recommendation: Restart SSPL service
2021-12-23 00:37:05 sspl [18089]: DEBUG [prepare_message] prepare_message, jsonMsg: {'username': 'SSPL-LL', 'description': 'Seagate Storage Platform Library - Sensor Response', 'title': 'SSPL Sensor Response', 'expires': -1, 'signature': 'N/A', 'time': '1640245025', 'message': {'sspl_ll_msg_header': {'schema_version': '1.0.0', 'sspl_version': '2.0.0', 'msg_version': '1.0.0'}, 'sensor_response_type': {'info': {'event_time': '1640245025', 'resource_id': 'MemFaultSensor', 'resource_type': 'node:sw:cortx_sw_services:sspl', 'description': "MemFaultSensor is stopped and unrecoverable. MemFaultSensor, 'SysFS' object has no attribute 'get_proc_memory'", 'impact': 'Server memory usage can not be monitored.', 'recommendation': 'Restart SSPL service', 'site_id': 'DC01', 'node_id': 'e33b735b62694a408dc082b0b671ba10', 'rack_id': 'RC01', 'cluster_id': '3e765bbb-8425-4e7f-8ee3-487e65ff2c8c', 'fru': 'false'}, 'alert_type': 'fault', 'severity': 'critical', 'specific_info': {'monitor': 'true', 'probe': 'sysfs', 'threaded': True}, 'alert_id': '1640245025f298a07009294b05b10380ae96303820', 'host_id': 'smc60-m11.colo.seagate.com'}}}
`